docs/blockchain-development-tutorials/forte/passkeys/index.md

```diff
@@ -205,15 +206,15 @@ const msgHex = encodeMessageFromSignable(signable, address)
 const payloadMsgHex = encodeTransactionPayload(signable.voucher)
 const role = msgHex === payloadMsgHex ? "payload" : "envelope"
 
-// 2) Compute SHA2-256(msgHex) -> 32-byte challenge (Flow keys commonly use SHA2_256)
+// 2) Compute SHA2-256(msgHex) -> 32-byte challenge
```

Comment:
This SHA2 used in the challenge building is not linked to the hashing attached to an account key. It is always SHA2 regardless of the account key hashing algo.